bartop-api/src/api/catalog/catalog.logic.js

```diff
@@ -0,0 +1,13 @@
+const findUser = require('../../utils/validators/userExists');
+
+module.exports = db => {
+  // create a new catalog
+  const create = async postBody => {
+    const { userId, drinkIds } = postBody;
+    const userToUpdate = await findUser(db, userId);
+    userToUpdate.catalog = drinkIds;
+    return await db.update('users', userId, userToUpdate);
```

Comment:
Is this where we'd change it to just do a `filter` and update the `user` without having to find first? Like

```javascript
r.table("posts").filter({auth0Id: userId}).update({catalog: drinkIds})
```
🤔

Reply:
Yes, exactly. PR for tearing out `js-data` should be inbound tonight or tomorrow. You can decide which one to review first, and I'll update the other to account for the changes